lib/td/command/workflow.rb

```diff
@@ -43,6 +43,9 @@ def workflow(op, capture_output=false, check_prereqs=true)
           ].join($/) + $/)
           cmd << '-Dio.digdag.standards.td.secrets.enabled=false'
           cmd << "-Dconfig.td.default_endpoint=#{Config.endpoint_domain}"
+          if workflow_endpoint.match(/\.connect\./i)
```

Comment:
Could you use the `match?` function? 
It will return `true` or `false` instead of `matching word` and is easier to match the logic.
`https://ruby-doc.org/core-2.5.1/Regexp.html#method-i-match-3F`

Reply:
Sorry my fault, the tests is running from some olders ruby version`2.1.x` while the `match?` is only available from `2.4` so please roll back to your origin propose.